api/src/services/users.js

```diff
@@ -50,7 +50,7 @@ const illegalDataModificationAttempts = data =>
 /*
  * Set error codes to 400 to minimize 500 errors and stacktraces in the logs.
  */
-const error400 = msg => ({ code: 400, message: msg });
+const error400 = (msg, key, params) => ({ code: 400, message: msg, translationKey: key, translationParams: params });
```

Comment:
It appears that you've added these new parameters (key and params) to this function which are never used, as you construct the `msg` object within each call. 
Adding the params is a good idea, but you need to use them in the calls, so not every call needs to know that your end response object should have `translationKey` and `translationParams` properties.